dnf-behave-tests/dnf/dnfdaemon/repoinfo.feature

```diff
@@ -0,0 +1,65 @@
+@not.with_os=rhel__ge__7
+@no_installroot
+
+Feature: Test dnfdaemon-client repoinfo
+
+Background: Using repositories dnf-ci-fedora and dnf-ci-thirdparty-updates
+  Given I use repository "dnf-ci-fedora"
+    And I use repository "dnf-ci-thirdparty-updates"
+    And I use repository "dnf-ci-fedora-updates" with configuration
+        | key     | value |
+        | enabled | 0     |
+    And I use repository "dnf-ci-thirdparty" with configuration
+        | key     | value |
+        | enabled | 0     |
+  Given I use repository "dnf-ci-fedora"
+    And I use repository "dnf-ci-fedora-updates"
+    And dnfdaemon-server is up and running
+
+Scenario: Call dnfdaemon-client repoinfo with no args
+   When I execute dnfdaemon-client with args "repoinfo"
+   Then the exit code is 0
+    And stdout matches line by line
+"""
+[0/0] dnf-ci-thirdparty-updates test re 100% | 530.5 KiB/s |   2.1 KiB |  00m00s
+[0/0] dnf-ci-fedora test repository     100% |   1.9 MiB/s |  94.0 KiB |  00m00s
+Repo ID             : dnf-ci-fedora
+Name                : dnf-ci-fedora test repository
+Status              : enabled
+Priority            :
+Cost                :
+Type                :
+Exclude packages    :
+Include packages    :
+Metadata expire     :
+Skip if unavailable :
+Config file         :
+URLs                :
+  Base URL          : file:///opt/ci/dnf-behave-tests/fixtures/repos/dnf-ci-fedora
+  Mirrorlist        :
+  Metalink          :
+GPG                 :
+  Keys              :
+  Verify repodata   :
+  Verify packages   :
+
+Repo ID             : dnf-ci-thirdparty-updates
+Name                : dnf-ci-thirdparty-updates test repository
+Status              : enabled
+Priority            :
+Cost                :
+Type                :
+Exclude packages    :
+Include packages    :
+Metadata expire     :
+Skip if unavailable :
+Config file         :
+URLs                :
+  Base URL          : file:///opt/ci/dnf-behave-tests/fixtures/repos/dnf-ci-thirdparty-updates
+  Mirrorlist        :
+  Metalink          :
+GPG                 :
+  Keys              :
+  Verify repodata   :
+  Verify packages   :
+"""
```

Comment:
I think we'll ideally want to reuse the existing scenarios for testing the daemon too, though how we deal with the different outputs is beyond me... (we need to discuss this)

Reply:
Well the daemon and microdnf are currently sharing some of outputs and the goal is to share all of them.